### ISSUE TITLE
DiCE doesn't change my outcome

### ISSUE THREAD

**Issue by ninaspreitzer** [interpretml/DiCE#294]
Hello, 

when I try to run DiCE, the proposed explanations do not change my outcome from 0 to . 
<img width="902" alt="Bildschirmfoto 2022-05-09 um 19 26 05" src="https://user-images.githubusercontent.com/57034840/167464266-e8b1c53d-94b6-4e66-9f73-97d16e045993.png">

I am only trying the basic settings with the adult dataset and didn't change anything of the code. 
any idea what I am doing wrong? 
Cheers

**Comment by amit-sharma** [comment]
That should not be happening. Can you share a working example of the exact code you are running to get this output? It will be helpful to debug. Are you following the getting started notebook?

**Comment by ninaspreitzer** [comment]
Hello amit, thank you so much for your answer! Yes I have followed the getting started and it is definitely odd as I remember that it once worked also for me.
<img width="902" alt="Bildschirmfoto 2022-05-09 um 18 05 54" src="https://user-images.githubusercontent.com/57034840/167579296-9e606390-8f7f-4b18-a6b7-c8a172b9ddff.png">
 
Some background maybe also doesn't hurt: I was trying to run the CARE framework for counterfactuals (see this repo: https://github.com/peymanrasouli/CARE) to compare it' output with DiCE. After some time I realized that the output of DiCE doesn't change anymore (not for ALL instances tho, but I would say 90% which doesn't make it easier to debug) 
To get to the root of the cause I wanted to only apply DiCE and now also have the same issue here.
